packages/core/tests/core.test.ts

```diff
@@ -295,6 +296,40 @@ describe('Core', () => {
           expect(plugin).toBeUndefined()
         })
       })
+
+      describe('startNetworkSpan', () => {
+        it('creates a network span', async () => {
+          const delivery = new InMemoryDelivery()
+          const client = createTestClient({ deliveryFactory: () => delivery })
+
+          client.start(VALID_API_KEY)
+
+          const span = client.startNetworkSpan({
+            method: 'GET',
+            url: 'https://example.com'
+          })
+
+          span.end({ status: 200 })
+
+          await jest.runOnlyPendingTimersAsync()
+
+          expect(delivery).toHaveSentSpan(expect.objectContaining({
+            name: '[HTTP]/GET',
+            kind: 3,
+            events: [],
+            spanId: 'a random 64 bit string',
+            traceId: 'a random 128 bit string',
+            startTimeUnixNano: expect.any(String),
+            endTimeUnixNano: expect.any(String)
```

Comment:
I think we'd expect page and referrer attributes on spans from browsers - do you think that's reasonably covered elsewhere to validate that it works for these manual spans or can we add an additional assertion for it?

Reply:
we have this covered already in the span-attributes-source browser tests and network-spans end to end tests for browser